codespell_lib/data/dictionary.txt

```diff
@@ -254,30 +273,41 @@ acqured->acquired
 acqures->acquires
 acquring->acquiring
 acrage->acreage
+acses->cases
 acssumed->assumed
+actally->actually
 actaul->actual
 actaully->actually
 actibity->activity
 actine->active
+actiual->actual
+activ->active
```

Comment:
not sure, even though this makes sense, there may be code that purposefully spells the word this way. So... it's debatable

Reply:
You tell me.

Reply:
Surely put it in, as it's a typo, and they should blacklist it if they want to purposefully misspell it?